Cloth.Domain/Cloth.Domain.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I don't seed here in PR but when I open the project see this:
![image](https://github.com/serjioGH/DigitallDotNetTask1/assets/8912816/983d9481-f272-4de6-a999-b856b0973f7b)

This should be removed.
Packages folder should be exclude from the solution